src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +28,13 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        
+        this.sequenceNumber= nextSequenceNumber;
+        nextSequenceNumber++;
+    }
+    
+    public void setSequenceNumber(int sequenceNumber){
+    	this.sequenceNumber = sequenceNumber;
     }
```

Comment:
Why are having a setter for this variable. Why should we allow this variable to be changed? Also can  sequenceNumber be declared final?